### DISCUSSION TITLE
Upgrade failed

### DISCUSSION THREAD

**Discussion by bpopovich44** [rancher/rke2#5777] [General]
I received this error while trying to install certs to rancher UI using command: 
helm upgrade  rancher rancher-stable/rancher --namespace cattle-system --set hostname=rancher.xxx.com --set replicas=3 --set ingress.tls.source=secret --set privateCA=true
UPGRADE FAILED: chart requires kubeVersion: < 1.27.0-0 which is incompatible with Kubernetes v1.27.10+rke2r1
I am currently on rke2 version 1.27.10+rke2r1.  Can this be fixed with updating helm or do I need to downgrade kube version?
To downgrade, would I run the install script but use a version?
Can you give me the commands for the master and worker nodes to downgrade, I don't see them posted.

Thank you.

**Comment by brandond** [comment]
1. Check the Rancher compatibility matrix. Kubernetes 1.27 should be supported by Rancher 2.8.
2. Downgrades are not supported.

**Reply by bpopovich44** [reply]
Thank you Brandon.  Looks like I'm on rancher 2.7.9.  So I need to upgrade Rancher through helm?  
https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades

**Reply by brandond** [reply]
Yep. rancher/rancher would be the correct place for this discussion though.

**Reply by bpopovich44** [reply]
Ok.  Thank you Brandon
